djimaging/tables/response/orientation.py

```diff
@@ -66,18 +64,22 @@ def get_time_dir_kernels(sorted_responses, dt):
     singular_value  float, 1st singular value
     """
 
-    U, S, V = np.linalg.svd(sorted_responses)
+    U, S, Vh = np.linalg.svd(sorted_responses)
 
     time_component = U[:, 0]
-    dir_component = V[0, :]
+    dir_component = Vh[0, :]
 
     # the time_kernel determined by SVD should be correlated to the average response across all directions. if the
     # correlation is negative, U is likely flipped
-    r, _ = stats.spearmanr(time_component, np.mean(sorted_responses, axis=-1), axis=1)
-    su = np.sign(r)
-    if su == 0:
+
+    if np.mean((-1 * time_component - np.mean(sorted_responses, axis=-1)) ** 2) < np.mean(
+        (time_component - np.mean(sorted_responses, axis=-1)) ** 2
+    ):
```

Comment:
This also now directly mimics how the sign logic was implemented in the original code.

Reply:
Same idea with different strategy (correlation bs MSE)